modules/friendlytag.js

```diff
@@ -189,10 +189,9 @@ Twinkle.tag.callback = function friendlytagCallback() {
 		case 'redirect':
 			Window.setTitle('Redirect tagging');
 
-			var i = 1;
 			$.each(Twinkle.tag.list.redirect, function(groupName, group) {
-				form.append({ type: 'header', id: 'tagHeader' + i, label: groupName });
-				var subdiv = form.append({ type: 'div', id: 'tagSubdiv' + i++ });
+				form.append({ type: 'header', label: groupName });
+				var subdiv = form.append({ type: 'div' });
```

Comment:
FWIW these two originate from 55c73041, likewise I don't think they were ever used, rather just there for the usefulness should someone want to reference them.